.github/workflows/cpaas.yml

```diff
@@ -105,7 +100,7 @@ jobs:
       rebuild-qa-containers: ${{ needs.init.outputs.rebuild-qa-containers == 'true' }}
     secrets: inherit
 
-  integration-tests:
+  amd64-integration-tests:
```

Comment:
There is some duplication here with `amd64-integration-tests`, `s390x-integration-tests`, and `ppc64le-integration-tests`. Perhaps this can be reduced.

Reply:
This is sort of intentionally duplicated to make it clearer which architecture tests are running.